Ginger/Ginger/Activities/ActivitiesListViewHelper.cs

```diff
@@ -479,6 +501,16 @@ public List<ListItemNotification> GetItemGroupNotificationsList(string GroupName
             {
                 List<ListItemNotification> notificationsList = new List<ListItemNotification>();
 
+                ListItemNotification publishInd = new ListItemNotification();
+                publishInd.AutomationID = "publishInd";
+                publishInd.ImageType = Amdocs.Ginger.Common.Enums.eImageType.Share;
+                publishInd.ToolTip = string.Format("{0} is marked to be Published to third party applications", GingerDicser.GetTermResValue(eTermResKey.ActivitiesGroup));
+                //publishInd.ImageForeground = Brushes.Orange;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4846856)